File: jambeth/jAmbeth.Merge/src/de/osthus/ambeth/merge/model/IDirectObjRef.java
package de.osthus.ambeth.merge.model;

import de.osthus.ambeth.annotation.XmlType;

@XmlType
public interface IDirectObjRef extends IObjRef
{
	Object getDirect();

	void setDirect(Object direct);

	int getCreateContainerIndex();

	void setCreateContainerIndex(int createContainerIndex);
}

File: ambeth/Ambeth.Merge/ambeth/merge/model/IDirectObjRef.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using De.Osthus.Ambeth.Util;
using De.Osthus.Ambeth.Annotation;

namespace De.Osthus.Ambeth.Merge.Model
{
    [XmlType]
    public interface IDirectObjRef : IObjRef
    {
        Object Direct { get; set; }

        int CreateContainerIndex { get; set; }
    }
}
